### ISSUE TITLE
2.0.5: Too many open files

### ISSUE THREAD

**Issue by silywalks** [mudape/iphonedetect#99]
After adding an iPhone which has been not yet seen by the tracker, lots of pipes are created until HA runs out of file handles after a few hours:
```
hass    5823 homeassistant   62w     FIFO               0,13       0t0   51588 pipe
hass    5823 homeassistant   63r     FIFO               0,13       0t0   51683 pipe
hass    5823 homeassistant   64r     FIFO               0,13       0t0   54686 pipe
hass    5823 homeassistant   65w     FIFO               0,13       0t0   51683 pipe
hass    5823 homeassistant   66r     FIFO               0,13       0t0   53833 pipe
hass    5823 homeassistant   67w     FIFO               0,13       0t0   53833 pipe
hass    5823 homeassistant   68r     FIFO               0,13       0t0   53962 pipe
```
`OSError: [Errno 24] Too many open files`
`Linux lubuntu-7650e8g 5.15.0-76-generic #83-Ubuntu SMP Thu Jun 15 19:16:32 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux`

Home Assistant 2023.7.2, python 3.10.6 and python 3.11.4

**Comment by mudape** [comment]
@silywalks ms1designs fixes are in 2.2.0, can you please try it?

**Comment by silywalks** [comment]
2.2.1 fixed it, thank you!
